eox_core/api/v1/views.py

```diff
@@ -835,6 +837,73 @@ def _section_breakdown(self, subsection_grades):
         return breakdown
 
 
+class EdxappUserCohort(UserQueryMixin, APIView):
+    """
+    Class used to GET a user's cohort.
+    """
+
+    authentication_classes = (BearerAuthentication, SessionAuthentication)
+    permission_classes = (IsAuthenticated,)
+    renderer_classes = (JSONRenderer, BrowsableAPIRenderer)
+
+    @apidocs.schema(
+        parameters=[
+            apidocs.query_parameter(
+                name="course_id",
+                param_type=str,
+                description="**required**, Used to identify the course where the cohort must be defined.",
+            ),
+            apidocs.query_parameter(
+                name="username",
+                param_type=str,
+                description="""**optional**, The username used to identify the user that belongs to the cohort.
```

Comment:
It looks like we can query for the cohorts of a user different than me, 

Can this have any security implications that we need to be aware about?

Reply:
I can't think of any security implications because we're not sharing critical data. Can you think of any?

I added this to be more flexible but right now I can't think of a specific use case for it, maybe it'd be better if we remove it. We could add it later if needed.

What do you think?

Reply:
The only thing that comes to mind is that you can confirm a username exists. But how far con you get from there, I'm not so sure. I would say if there isn't any clear use case for now just remove it. If there is any need for that functionality in the future you can add it without breaking backwards compatibility.